pkg/volume/azure_file/azure_file.go

```diff
@@ -0,0 +1,236 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package azure_file
+
```

Comment:
@saad-ali given that provisioning is WIP by @markturansky , and that Azure storage setup is different from AWS/GCE/OpenStack, I'd wait till provisioner is able to cope with these differences. 

Thought?